src/DefaultRenderer.js

```diff
@@ -212,6 +212,15 @@ export default class DefaultRenderer extends Component {
     }
 
     if (SceneComponent) {
+      if (navigationState.tabs || ! navigationState.parent) {
```

Comment:
I'm wondering why you would need this. navigationState.tabs will never have a component attached to it and this condition is for navigationState.component, or maybe i got it wrong. 
<Scene key="my_tabs" tabs>
 .... your other scenes with components that represent tab views ... 
</Scene>